app/services/hts_service/ait_intergration/contact_csv_row_builder.rb

```diff
@@ -10,7 +10,15 @@ def last_name contact, index
     end
 
     def sex contact, index
```

Comment:
Why are we maintaining the index argument

Reply:
it is an index as in contact[0], contact[1]..   not the index client (naming issue)